### ISSUE TITLE
Tutorial example doesn't make sense

### ISSUE THREAD

**Issue by calebweinreb** [rly/ndx-pose#16]
There are a few things in the tutorial example that don't really make sense for multi-camera data.

- The line `confidence_definition='Softmax output of the deep neural network.'` only makes sense for single-camera. For 3D keypoints inferred with DLC, there would be separate such softmax outputs for each camera, and the value going into NWB (if there were one) would be some custom confidence score output by the triangulation method.
- The unit `unit='pixels'` would be typical of 2D data, but 3D data would usually have real-world units inferred through calibration (e.g. mm)
- The line `dimensions=np.array([[640, 480], [1024, 768]], dtype='uint8'),` only makes sense for 2D data. For 3D, this might differ across cameras.
